cmd/ci-operator/main.go

```diff
@@ -534,6 +537,31 @@ func (o *options) initializeNamespace() error {
 		break
 	}
 
+	if o.givePrAuthorAccessToNamespace {
+		// Generate rolebinding for the PR Author.
+		if len(o.jobSpec.Refs.Pulls) == 1 {
```

Comment:
```go
for _, pull := range o.jobSpec.Refs.Pulls {
```

Reply:
In which cases there are more than 1 PRs ? Only in batch jobs right. I don't think that we need to include this case as well.

Reply:
I don't see a great reason not to allow access to all authors in a batch. If the job is still being tested, we might want to investigate a batch run.

Reply:
This is a nit, if you feel strongly on it we can merge as-is